--- ansible/kolla-venv/bin/kolla-ansible.orig	2017-01-29 21:20:29.189225104 +0000
+++ ansible/kolla-venv/bin/kolla-ansible	2017-01-31 16:31:07.203695865 +0000
@@ -9,6 +9,8 @@
         BASEDIR=/usr/share/kolla
     elif [[ ${dir_name} == "/usr/local/bin" ]]; then
         BASEDIR=/usr/local/share/kolla
+    elif [[ ${dir_name} == "${VIRTUAL_ENV}/bin" ]]; then
+        BASEDIR="${VIRTUAL_ENV}/share/kolla"
     else
         BASEDIR="$(dirname ${dir_name})"
     fi
